src/renderers/dom/shared/ReactDOMComponent.js

```diff
@@ -687,6 +698,19 @@ ReactDOMComponent.Mixin = {
   },
 
   /**
+   * Setup this component to trap non-bubbling events locally
+   *
+   * @private
+   * @param {ReactReconcileTransaction|ReactServerRenderingTransaction} transaction
+   */
+  _setupTrapBubbledEventsLocal: function(transaction) {
+    this._wrapperState = {
+      listeners: null,
+    };
+    transaction.getReactMountReady().enqueue(trapBubbledEventsLocal, this);
```

Comment:
I know it's merged. I was hoping that one of you would want to post a PR to fix it but I did it myself in #5213.